src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,75 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev2 (02/27/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.5. (#9187)
```

Comment:
In the email, I think we should mention the fix for Python 2 users getting a Python 2 EOL deprecation warning every time they run Pex.